internal/model/user_server.go

```diff
@@ -160,10 +160,22 @@ func (this *ServerUser) GetCourseRole(courseID string) CourseUserRole {
 }
 
 // Convert this server user into a course user for the specific course.
+// Set courseOnly to false if high level server users should be converted to a course user.
 // Will return (nil, nil) if the user is not enrolled in the given course.
-func (this *ServerUser) ToCourseUser(courseID string) (*CourseUser, error) {
+func (this *ServerUser) ToCourseUser(courseID string, courseOnly bool) (*CourseUser, error) {
 	info, exists := this.CourseInfo[courseID]
```

Comment:
Swap out `exists` with `enrolled`.
`exists` is fine for general cases, but we are checking something specific in this case.